client/src/pages/customer-pages/FlightMap/index.js

```diff
@@ -62,6 +71,16 @@ class FlightMap extends Component {
           &nbsp;-&nbsp;
           <label>{destination}</label>
         </div>
+        <Map
+          markers={[
+            { position: positions[origin] },
+            { position: positions[destination] }
+          ]}
+          zoom={5}
+          center={{ lat: 9.072264, lng: 7.491302 }}
+          containerElement={<div style={{ height: 100, width: 300 }} />}
+          mapElement={<div style={{ height: 230, width: 300 }} />}
```

Comment:
Just for the sake of consistency with the way you apply styles to the rest of the app, you could give these divs classNames and put the sizes into your css file rather than having the inline styles.  It's not incredibly important though 😀